parlai/scripts/distributed_train.py

```diff
@@ -52,8 +53,10 @@ def main():
     parser = single_train.setup_args()
     parser.add_distributed_training_args()
     parser.add_argument('--port', type=int, default=61337, help='TCP port number')
-    opt = parser.parse_args(print_args=(os.environ['SLURM_PROCID'] == '0'))
+    return parser
 
+
+def dist_train(opt):
```

Comment:
the node_list lint error is a bug

Reply:
I see, regardless, I am not sure why the lint and 3 other tests didn't run on latest commit.

Does the rest of the PR look good ?

Reply:
man what is wrong with your account and CI...

You can run `bash autoformat.sh` to get lint locally

Reply:
spot check they look fine but lint was definitely finding bugs from copy paste failures

Reply:
Output from ```bash autoformat.sh```

```
All done! ✨ 🍰 ✨
18 files left unchanged.
```